Code/GraphMol/FileParsers/testPropertyLists.cpp

```diff
@@ -150,7 +150,7 @@ M  END
 >  <atom.iprop.NumHeavyNeighbors>  (1) 
 2 2 2
 
->  <atom.prop.AtomLabel>  (1) 
+>  <atom.prop.AtomSymbol>  (1)
```

Comment:
Is this a result of automatic refactoring gone wrong?

Reply:
I would imagine so.  I did call it AtomLabel to begin with, and then did a re-factor.  It didn't ever put anything in that file or the one below, though.